build/yaml/jobs/binaries.yaml

```diff
@@ -19,8 +19,9 @@ jobs:
       CopyFilters: |
         **
         !**\Nagler*
-        !**\*Tests*
-        !**\*Tests*\**
+        !**\*Test*
+        !**\*Test*\**
+      PublishSymbols: true
```

Comment:
>PublishSymbols: true [](start = 6, length = 20)

I'll look into adding this for Linux as well.

Reply:
Does this mean that we will be publishing symbols for all builds? Even PRs and CIs? Seems like we should only be doing it for the real signed build.

---
In reply to: [333181904](https://github.com/microsoft/CLRInstrumentationEngine/pull/134#discussion_r333181904) [](ancestors = 333181904)

Reply:
No, this is only for when SignType is set (to Test or Real). I can make it strictly for Real signed.

---
In reply to: [333184937](https://github.com/microsoft/CLRInstrumentationEngine/pull/134#discussion_r333184937) [](ancestors = 333184937,333181904)

Reply:
Didn't see the if statement above.

---
In reply to: [333184937](https://github.com/microsoft/CLRInstrumentationEngine/pull/134#discussion_r333184937) [](ancestors = 333184937,333181904)

Reply:
I'll add more comments here for clarity

---
In reply to: [333216207](https://github.com/microsoft/CLRInstrumentationEngine/pull/134#discussion_r333216207) [](ancestors = 333216207,333184937,333181904)

Reply:
Regarding the Linux symbols, I'll hold off as Aerobee doesn't do this either at the moment.

---
In reply to: [333242781](https://github.com/microsoft/CLRInstrumentationEngine/pull/134#discussion_r333242781) [](ancestors = 333242781,333216207,333184937,333181904)